flow/connectors/postgres/qrep_sync_method.go

```diff
@@ -128,16 +142,19 @@ func (s *QRepStagingTableSync) SyncQRepRecords(
 			}
 			selectStrArray = append(selectStrArray, fmt.Sprintf(`"%s"`, col))
 		}
-
+		setClauseArray = append(setClauseArray,
+			fmt.Sprintf(`"%s" = CURRENT_TIMESTAMP`, syncedAtCol))
 		setClause := strings.Join(setClauseArray, ",")
-		selectStr := strings.Join(selectStrArray, ",")
+		selectStrColsSQL := strings.Join(append(selectStrArray,
+			fmt.Sprintf(`"%s"`, syncedAtCol)), ",")
+		selectStrValuesSQL := strings.Join(append(selectStrArray, `CURRENT_TIMESTAMP`), ",")
 
 		// Step 2.3: Perform the upsert operation, ON CONFLICT UPDATE
 		upsertStmt := fmt.Sprintf(
 			"INSERT INTO %s (%s) SELECT %s FROM %s ON CONFLICT (%s) DO UPDATE SET %s;",
```

Comment:
```suggestion
			"INSERT INTO %s (%s, "%s") SELECT %s, CURRENT_TIMESTAMP FROM %s ON CONFLICT (%s) DO UPDATE SET %s;",
```
& then pass `syncedAtCol` as an extra format param. Avoids aliased appends